phpcs.xml.dist

```diff
@@ -342,4 +342,6 @@
 		<exclude-pattern>/src/wp-content/themes/twentyfourteen/inc/widgets\.php</exclude-pattern>
 		<exclude-pattern>/src/wp-content/themes/twentyfourteen/inc/featured-content\.php</exclude-pattern>
 	</rule>
+
+	<rule ref="SlevomatCodingStandard.Functions.StaticClosure" />
```

Comment:
This needs to be removed.

It is not a rule in the official coding standards, so, while I agree, it is a good "best practice", we cannot demand for it to be enforced.